metadata-ingestion/docs/sources/vertica/README.md

```diff
@@ -3,9 +3,13 @@
 <!-- Plain-language description of what this integration is meant to do.  -->
 <!-- Include details about where metadata is extracted from (ie. logs, source API, manifest, etc.)   -->
 
-This plugin extracts metadata for Tables and Views on Vertica.
+The DataHub Vertica Plugin extracts the following: 
+ 
+* Metadata for databases, schemas, views, tables, and projections
+* Table level lineage
+* Metadata for ML Models
+* Metadata for Vertica OAuth
```

Comment:
Curious how we model this? What counts as metadata for vertica oauth?

Reply:
Metadata for oauth is information about the oauth user . There are 7 different type of metadata we collect which is shows in UI under a seperate entities section .

Reply:
Which section? I'm not familiar

Reply:
![image](https://user-images.githubusercontent.com/110387730/212102363-2ea71ac8-b7bb-4c34-8f0c-1618ed62185d.png)

Reply:
![Brush-#2](https://user-images.githubusercontent.com/110387730/212103359-06e9ac35-62ce-4228-9f63-3773e74629b6.png)

Reply:
This is how OAuth metadata is shown, this was part of the last PR.